src/components/MoneyRequestConfirmationList.js

```diff
@@ -290,15 +325,23 @@ function MoneyRequestConfirmationList(props) {
             optionHoveredStyle={canModifyParticipants ? styles.hoveredComponentBG : {}}
             footerContent={footerContent}
         >
-            <MenuItemWithTopDescription
-                shouldShowRightIcon={!props.isReadOnly}
-                title={formattedAmount}
-                description={translate('iou.amount')}
-                onPress={() => Navigation.navigate(ROUTES.getMoneyRequestAmountRoute(props.iouType, props.reportID))}
-                style={[styles.moneyRequestMenuItem, styles.mt2]}
-                titleStyle={styles.moneyRequestConfirmationAmount}
-                disabled={didConfirm || props.isReadOnly}
-            />
+            {props.receiptPath && (
```

Comment:
From our [style guide](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#important-note)...

> In React Native, one must not attempt to falsey-check a string for an inline ternary. Even if it's in curly braces, React Native will try to render it as a <Text> node and most likely throw an error about trying to render text outside of a <Text> component. Use _.isEmpty() instead.

```suggestion
            {!_.isEmpty(props.receiptPath) && (
```

Reply:
Great to know 👍